cni/Dockerfile

```diff
@@ -1,13 +1,15 @@
+# !! AUTOGENERATED - DO NOT EDIT !!
```

Comment:
Is there any way we can reject commits that touch this by hand? I suppose this applies to auto-generated things in general, but habits are so ingrained w/r/t Dockerfile edits that it might be more of a hazard here.

Reply:
that will happen implicitly because the pipeline added here generates and then checks for changes. so if you edit the Dockerfiles directly the pipeline will clobber your changes and the tree will be dirty